hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1696,4 +1697,37 @@ enum ResponseCodeEnum {
      * Max custom fees list is not supported for this operation.
      */
     MAX_CUSTOM_FEES_IS_NOT_SUPPORTED = 387;
+
+    /**
+    * The list of batch transactions is empty
+    */
+    BATCH_LIST_EMPTY = 388;
+
+    /**
+    * The list of batch transactions contains duplicated transactions
+    */
+    BATCH_LIST_CONTAINS_DUPLICATES = 389;
+
+    /**
+    * The list of batch transactions contains a disallowed transaction type.
+    */
+    BATCH_LIST_CONTAINS_INVALID_TRANSACTION = 390;
```

Comment:
`BATCH_LIST_CONTAINS_INVALID_TRANSACTION` , this is not invalid transaction, but not supported transaction.